src/symtab2gb/symtab2gb_parse_options.cpp

```diff
@@ -111,6 +111,7 @@ int symtab2gb_parse_optionst::doit()
   {
     gb_filename = cmdline.get_value(SYMTAB2GB_OUT_FILE_OPT);
   }
+  register_language(new_json_symtab_language);
```

Comment:
Failing build note: `symtab2gb_parse_options.cpp:114:3: error: ‘register_language’ was not declared in this scope`

Reply:
Ooops.  Should be fixed now.